collectors/auth0/cfn/auth0-collector.template

```diff
@@ -87,7 +87,7 @@
           "PawsCollectorTypeName" : "auth0",
           "AlertlogicCustomerId" : { "Ref":"AlertlogicCustomerId" },
           "CollectorId" : { "Ref":"CollectorId" },
-          "PollingInterval" : 60,
+          "PollingInterval" : 600,
```

Comment:
I know i asked for this to be increased. but I think we can bring is back down to 60, given what we've figured about what the problem is.

Reply:
Logs are continuously coming, I'm not getting any condition where logs are not present in realtime. Because of this, the collector is running continuously with poll_interval_seconds 1, so I have changed poll_interval 60 to 600 in CFN.

Reply:
so it'll wait for 10 min in real-time

Reply:
Thats true with our test account with a lot of logs coming in continuously. But this value will kick in when the collector is caught up. I think waiting 10 mins for logs to build up might be too long. We would like logs to come in as soon as they can.

Reply:
Right